react/components/ProgressBar/utils.ts

```diff
@@ -35,11 +36,18 @@ export function generateProgressBarStates(
   })
 }
 
+export function getCurrentProgressBarState(status: string, packages: any) {
+  const currentProgressIndex = getOrderProgress(status, packages)
+
+  return generateProgressBarStates(progressBarStates, currentProgressIndex, packages)[currentProgressIndex]?.label
```

Comment:
probably nit, but better reading (debatable though):
```suggestion
  const progressBarStates = generateProgressBarStates(progressBarStates, currentProgressIndex, packages)

  return progressBarStates[currentProgressIndex]?.label
```

Reply:
I agree storing in a variable looks better!